cpp/log/etcd_consistent_store.h

```diff
@@ -71,6 +71,11 @@ class EtcdConsistentStore : public ConsistentStore<Logged> {
   template <class T>
   util::Status ForceSetEntry(const std::string& path, EntryHandle<T>* entry);
 
+  template <class T>
+  util::Status ForceSetEntryWithTTL(const std::string& path,
+                                    const std::chrono::duration<int>& ttl,
```

Comment:
I'd prefer using the ```std::chrono::seconds``` typedef, doesn't really change anything, but I find it nicer to read.